airflow-core/src/airflow/ui/src/i18n/config.ts

```diff
@@ -28,6 +28,7 @@ export const supportedLanguages = [
   { code: "es", name: "Español" },
   { code: "fr", name: "Français" },
   { code: "he", name: "עברית" },
+  { code: "hu", name: "Hungarian" },
```

Comment:
This is the local representation of the language... where "Hungarian" is the english term, would "Magyar" be correct?
```suggestion
  { code: "hu", name: "Magyar" },
```

Reply:
Yes, I will correct that